bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx83.html

```diff
@@ -0,0 +1,82 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_83') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md wnp83-hero-logo">
+      {% include 'firefox/whatsnew/includes/fx83/pocket.svg' %}
+      <h1 class="wnp-main-title">{{ ftl('whatsnew83-main-title') }}</h1>
+      <p class="wnp-main-tagline">{{ ftl('whatsnew83-main-intro') }}</p>
+
+      <div class="wnp-main-cta">
+        <a href="https://getpocket.com/explore?utm_source=mozilla.org-whatsnew83&utm_medium=referral&utm_campaign=whatsnew83&entrypoint=mozilla.org-whatsnew83" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN Now" data-cta-type="button">{{ ftl('whatsnew83-main-cta') }}</a>
```

Comment:
```suggestion
        <a href="https://getpocket.com/explore?utm_source=mozilla.org-whatsnew83&utm_medium=referral&utm_campaign=whatsnew83&entrypoint=mozilla.org-whatsnew83" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Pocket Now" data-cta-type="button">{{ ftl('whatsnew83-main-cta') }}</a>
```

Reply:
Do I need to use the `ftl` entry, or can it be in plain English?

Reply:
Plain English for these. 

When GTM takes these and puts them in GA this value becomes the event label. Setting this value in a data attribute means two things: 1) we can tweak the text without affecting the tracking 2) we can compare performance across locales using the same event labels.

(I think it's event lable - could be event action)

Reply:
👍